Application/EdFi.Ods.Extensions.TPDM/Artifacts/MsSql/Structure/Ods/0040-EXTENSION-TPDM-IdColumnUniqueIndexes.sql

```diff
@@ -1,8 +1,3 @@
--- SPDX-License-Identifier: Apache-2.0
```

Comment:
we need to keep these license headers

Reply:
at least according to the docs:
https://techdocs.ed-fi.org/display/EDFIODS/License+and+Copyright+Notifications
https://techdocs.ed-fi.org/display/EDFIODS/Analysis+Spike%3A+Tool+Support+for+File+Headers

Reply:
license headers are added